<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-2</y>
      <width>1280</width>
      <height>731</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPage">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>150</y>
      <width>135</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>12</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPage_3">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>230</y>
      <width>135</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>12</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Calendar</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPage_4">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>310</y>
      <width>135</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>12</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Converter</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPage_5">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>470</y>
      <width>135</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>12</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="digitalTimer">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Clock</string>
    </property>
   </widget>
   <widget class="QLabel" name="showDate">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPage_6">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>390</y>
      <width>135</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>12</pointsize>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>TodoList</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="weather" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>800</width>
      <height>210</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(24, 24, 24);
	qproperty-alignment: AlignCenter;
}
</string>
    </property>
    <widget class="QLabel" name="title_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt</string>
     </property>
     <property name="text">
      <string>Weather Forcast</string>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>761</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="temperature_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="temp1_4">
        <property name="text">
         <string>19 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp2_4">
        <property name="text">
         <string>21 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp3_4">
        <property name="text">
         <string>23 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp4_4">
        <property name="text">
         <string>25 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp5_4">
        <property name="text">
         <string>27 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp6_4">
        <property name="text">
         <string>26 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp7_4">
        <property name="text">
         <string>24 ˚C</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="temp8_4">
        <property name="text">
         <string>22 ˚C</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>761</width>
       <height>20</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="time_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="time1_4">
        <property name="text">
         <string>9 AM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time2_4">
        <property name="text">
         <string>10 AM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time3_4">
        <property name="text">
         <string>11 AM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time4_4">
        <property name="text">
         <string>12 AM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time5_4">
        <property name="text">
         <string>1 PM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time6_4">
        <property name="text">
         <string>2 PM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time7_4">
        <property name="text">
         <string>3 PM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time8_4">
        <property name="text">
         <string>4 PM</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="widget_9" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>771</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: rgba(60, 60, 60, 100);
	border-radius: 4px;
}</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QLabel" name="lblTypeIcon2_13">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/DuoYun.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="lblTypeIcon3_4">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/DuoYun.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="lblTypeIcon2_14">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/DuoYun.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lblTypeIcon0_4">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/Qing.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="lblTypeIcon2_15">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/DuoYun.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="lblTypeIcon2_16">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/DuoYun.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblTypeIcon1_4">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/XiaoYu.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="lblTypeIcon4_4">
        <property name="styleSheet">
         <string notr="true">border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>:/res/type/ZhongYu.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="frame_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>801</width>
       <height>211</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
	border-radius: 20px;
	background-color: white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <zorder>frame_10</zorder>
    <zorder>title_4</zorder>
    <zorder>horizontalLayoutWidget_7</zorder>
    <zorder>horizontalLayoutWidget_8</zorder>
    <zorder>widget_9</zorder>
   </widget>
   <widget class="QWidget" name="quote" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="frame_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
	border-radius: 20px;
	background-color: white;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="quoteTitle">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
qproperty-alignment: AlignCenter;
color:rgb(255, 137, 106)</string>
      </property>
      <property name="text">
       <string>Quote of the Day</string>
      </property>
     </widget>
     <widget class="QLabel" name="quoteBar">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>291</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(44, 175, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="notification" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QLabel {
	color:rgb(255, 137, 106);
	qproperty-alignment: AlignCenter;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="cityName">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="temp">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>181</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="msg">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>29</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>340</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="min">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>340</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="descrip">
    <property name="geometry">
     <rect>
      <x>526</x>
      <y>380</y>
      <width>190</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>27</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>130</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>100</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select a city: </string>
    </property>
   </widget>
   <zorder>label_pic</zorder>
   <zorder>btnPage</zorder>
   <zorder>btnPage_3</zorder>
   <zorder>btnPage_4</zorder>
   <zorder>btnPage_5</zorder>
   <zorder>digitalTimer</zorder>
   <zorder>btnPage_6</zorder>
   <zorder>weather</zorder>
   <zorder>quote</zorder>
   <zorder>notification</zorder>
   <zorder>label</zorder>
   <zorder>temp</zorder>
   <zorder>showDate</zorder>
   <zorder>descrip</zorder>
   <zorder>msg</zorder>
   <zorder>max</zorder>
   <zorder>cityName</zorder>
   <zorder>min</zorder>
   <zorder>comboBox</zorder>
   <zorder>label_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
